rotkehlchen/externalapis/opensea.py

```diff
@@ -140,12 +136,14 @@ def _query(
             self.session.headers.update({'X-API-KEY': api_key})
 
         if endpoint == 'collectionstats':
-            query_str = f'https://api.opensea.io/api/v1/collection/{options["name"]}/stats'  # type: ignore
+            query_str = f'https://api.opensea.io/api/v2/collections/{options["name"]}/stats'  # type: ignore
+        elif endpoint == 'collection':
+            query_str = f'https://api.opensea.io/api/v2/collections/{options["collection_slug"]}'  # type: ignore
         elif endpoint == 'asset':
-            query_str = f'https://api.opensea.io/api/v1/asset/{options["address"]}/{options["item_id"]}'  # type: ignore
+            query_str = f'https://api.opensea.io/api/v2/chain/ethereum/contract/{options["address"]}/nfts/{options["item_id"]}'  # type: ignore
             options = None
         else:
-            query_str = f'https://api.opensea.io/api/v1/{endpoint}'
+            query_str = f'https://api.opensea.io/api/v2/chain/ethereum/account/{options["address"]}/nfts'  # type: ignore
```

Comment:
Something seems off here.

The possible endpoints are given in the literal above `'assets', 'collections', 'collectionstats', 'asset', 'collection'`.

Then at the if elif/else we are missing `collections` and `assets`. So the `else` case can't be both.

Reply:
Removed 'collections' as it's not relevant anymore.